pymc/distributions/multivariate.py

```diff
@@ -2061,7 +2061,10 @@ def make_node(self, rng, size, dtype, mu, W, alpha, tau):
             W = Assert(msg)(W, pt.allclose(W, W.T))
 
         tau = pt.as_tensor_variable(floatX(tau))
+
         alpha = pt.as_tensor_variable(floatX(alpha))
+        msg = "the domain of alpha is: -1 < alpha < 1."
+        alpha = Assert(msg)(alpha, pt.lt(alpha, 1) and pt.gt(alpha, -1))
```

Comment:
You shouldn't put this assert here. Instead raise an error in the `rng_fn` if the alpha is actually problematic for random draws.

Otherwise Asserts will complicate the graph needlessly.

Anyway the reason you don't see the exception is probably that you need to use `pt.and_` instead of python `and`.

Reply:
Yup the `pt.and_` was what I was missing. So instead of using assert in the rng function, I just raise a ValueError? Is that right?

    if alpha >= 1 or alpha <= -1:
                raise ValueError("the domain of alpha is: -1 < alpha < 1")

Reply:
Yes, but alpha could be a vector so you need a `np.all(...)`

Reply:
gotcha - okay sent in a new PR